src/wp-includes/class-wp-textdomain-registry.php

```diff
@@ -84,7 +93,11 @@ public function get( $domain, $locale ) {
 	 * @return bool Whether any MO file paths are available for the domain.
 	 */
 	public function has( $domain ) {
-		return ! empty( $this->current[ $domain ] ) || empty( $this->all[ $domain ] );
+		return (
+			! empty( $this->current[ $domain ] ) ||
+			empty( $this->all[ $domain ] ) ||
+			in_array( $domain, $this->domains_with_translations, true )
```

Comment:
I don't think this works as expected because `$this->domains_with_translations` is an array of booleans keyed by the domain string. 

```suggestion
			isset( $this->domains_with_translations[ $domain ] )
```

Reply:
Ah yes, I changed this structure in my last commit